src/modules/freia/geometry/Config.h

```diff
@@ -42,6 +42,9 @@ class Config : public VMM3Config {
   bool WireGapCheck{true};
   uint16_t MaxGapWire{0};
   uint16_t MaxGapStrip{0};
+  bool SplitMultiEvents{false};
+  float SplitMultiEventsCoefficientLow{0.8};
```

Comment:
This should probably be a single tolerance value.

Reply:
I based this off Francesco's Freia multi-hit code where he has an option for a different low and high coefficient. There are also often situations where they won't centre around 1, where strips and wires have very different sensitivities, if I remember right the coefficients for freia are generally around 2.